README.md

```diff
@@ -39,7 +39,8 @@ For more information please check out the following guides.
 
 ## Information for Contributors
 
-* [Developer Guide][developer-guide]
+* [General Developer Guide for Import-Export UDF][developer-guide]
```

Comment:
I could not find file "developer_guide":

Reply:
It is at the end of the file:
```
[developer-guide]: https://github.com/exasol/import-export-udf-common-scala/blob/master/doc/development/developer_guide.md
```